frontend/app/component/form/voters_and_votes/VotersAndVotesForm.tsx

```diff
@@ -222,7 +239,53 @@ export function VotersAndVotesForm() {
             format={format}
             defaultValue={sectionValues.votes_counts.total_votes_cast_count}
             isTotal
+            addSeparator={recounted}
           />
+          {recounted && (
+            <>
+              <InputGridRow
+                key="A.2"
+                field="A.2"
+                id="poll_card_recount"
+                title="Stempassen (na hertelling door gemeentelijke stembureau)"
```

Comment:
@Lionqueen94 I've made a small tweak to the design to prevent the repetition of labels. Hope you don't mind. See [Figma](https://www.figma.com/design/zZlFr8tYiRyp4I26sh6eqp/Kiesraad---Abacus-optelsoftware?node-id=3315-15053&t=Se55Zt4P8UonnJcZ-4).

Reply:
I've removed the label suffix in aa75f9584074e039ee4780e79a71656335436ada.